src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -203,4 +204,46 @@ public String validateGiverRecipientVisibility(FeedbackQuestionAttributes feedba
         return "";
     }
 
+    @Override
+    public String getJsonString() {
+        if (isEmptyRecommendedLength()) {
```

Comment:
I don't quite understand why we need the special check for `isEmptyRecommendedLength()` here. What happen if we just use `super.getJsonString();`?

Reply:
If a FeedbackTextQuestion with plain text in data store got updated and saved back to data store, it will not be plain text anymore with `super.getJsonString()`. Or, do we allow such migration happen in data store?

Reply:
> it will not be plain text anymore

It is fine. In fact, this is the existing way in V6/7.